app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,19 @@
 <resources>
     <string name="app_name">CompareRxJava1VsRxJava2</string>
+    <string name="buffer">Buffer</string>
+    <string name="flatmap">FlatMap</string>
+    <string name="group_by">GroupBy</string>
+    <string name="map">Map</string>
+    <string name="scan">Scan</string>
+    <string name="window">Window</string>
+    <string name="transforming_operators">Transforming Operators</string>
+    <string name="filtering_observables">Filtering Observables</string>
+    <string name="combining_observables">Combining Observables</string>
+    <string name="error_handling_operators">Error Handling Operators</string>
+    <string name="observable_utility_operators">Observable Utility Operators</string>
+    <string name="conditional_and_boolean_operators">Conditional and Boolean Operators</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.conditional_and_boolean_operators` appears to be unused
 Unused resources make applications larger and slow down builds.